app/dfda/components/CitizenScienceSection.tsx

```
@@ -0,0 +1,59 @@
+import { Microscope, Globe, Search } from 'lucide-react'
+import { FeatureBox } from './FeatureBox'
+
+export default function CitizenScienceSection() {
+  const features = [
+    {
+      title: "Create Your Own Studies",
+      desc: "Anyone can create a study, become a prestigious scientist, get a link, and invite all their friends to join!",
+      color: "bg-purple-400",
+      icon: Microscope,
+      media: "https://fdai.earth/wp-content/uploads/2024/03/create-study.gif",
+      onClick: () => window.location.href = "https://safe.fdai.earth/app/public/#/app/study-creation"
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)